src/data/MTRStationData.tsx

```diff
@@ -0,0 +1,37 @@
+export interface MTRStation {
+  id: string;
+  name: string;
+  stations: {id: string, name: string}[]
+}
+
+const mtrStations: MTRStation[] = ["AEL Airport Express HOK Hong Kong KOW Kowloon TSY Tsing Yi AIR Airport AWE AsiaWorld Expo",
```

Comment:
Can we also add chinese?

Reply:
Please create an issue for UX changes as Chinese station names are not included in their API documentations, functionality wise we can go on :D